decentraland/unity-renderer
# feat: show alice tutorial on deep link

## What does this PR change?

Changes which tutorial is shown after entering Decentraland for the first time using deep link. Instead of showing the canvas with the key references, it now shows the Alice tutorial.



## How to test the changes?

1. Enter a fresh guest version without cookies to https://play.decentraland.zone/?renderer-branch=feat/show-alice-tutorial-on-deep-link&position=0,0. The Alice tutorial should appear, starting with the "Welcome message". There should be tutorial music playing. Music should fadeout on exiting the tutorial.
2. Enter a fresh guest version without cookies to https://play.decentraland.zone/?renderer-branch=feat/show-alice-tutorial-on-deep-link&position=100,100. The Alice tutorial should appear from the lock cursor step, no "Welcome message" should appear. There should not be tutorial music playing.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
